packages/plugins/ui-theme/src/store/store.ts

```diff
@@ -0,0 +1,16 @@
+import { init, RematchDispatch, RematchRootState } from '@rematch/core';
+import loadingPlugin, { ExtraModelsFromLoading } from '@rematch/loading';
+
+import { models, RootModel } from './models';
+
+type FullModel = ExtraModelsFromLoading<RootModel>;
+
+export const store = init({
```

Comment:
to avoid the @ts-ignore you can pass 
```suggestion
export const store = init<RootModel, FullModel>({
```